### DISCUSSION TITLE
I Can See my Boiler Information, but What Now?

### DISCUSSION THREAD

**Discussion by CharlesAlvis** [emsesp/EMS-ESP32#1401] [General]
Hello Fellow EMS-ESP32 Users,

I hope you're all doing well. I wanted to reach out and share a situation I'm currently facing with a property in the United States. You see, boilers aren't very popular here, but the property we acquired about a year ago uses radiant heating and on-demand hot water. Since this property isn't our primary residence, it's crucial for me to ensure everything is functioning properly, especially during freezing temperatures.  The boiler brand that I have is a Buderus.

Now, the reason I'm reaching out is because I'm looking for some guidance on how to receive notifications for select changes in the boiler operation. I've identified two key pieces of information that are important to me to monitor: a change in the EMS-ESP **Maintenance Message** code and a change in **System Pressure**. I believe having triggers and notifications in place is essential in most cases, so I'm eager to explore the best options available.

If any of you have experience or knowledge in this area, I would greatly appreciate your guidance and expertise. Your insights could make a significant difference in ensuring the smooth operation of this property. Thank you in advance for your assistance, and I look forward to hearing from you soon.   I should point out I am using Home Assistant.

Best regards,
CA

**Comment by proddy** [comment]
Remote access is tricky. I see two ways
 - create a VPN tunnel between the property's router and the internet so you can use the API to remotely poll the data, and create a script to notify on any changes (error codes, pressure etc)
 - use a remote public MQTT broker and connect EMS-ESP to that (something cheap with low data retention). You'll only need to open up one SSL port. Then from your home connect an automation app to it, like Home Assistance. 

@bbqkees ?
